code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -14,13 +14,11 @@ bzformation = 4
 freonformation = 5
 stimformation = 6
 nobliumformation = 7
-hexane_plasma_filtering = 8
 zauker_decomp = 9
 healium_formation = 10
 proto_nitrate_formation = 11
 zauker_formation = 12
 halon_formation = 13
-hexane_formation = 14
 proto_nitrate_response = 15 - 18
 fusion = 19
 metallic_hydrogen = 20
```

Comment:
Move over the priorities so that everything stays sequential.

Reply:
In a more perfect world priorities would be cascading defines, but we can deal with that later